<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropboxFabaria</class>
 <widget class="QDialog" name="DropboxFabaria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>320</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>575</width>
    <height>320</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>575</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backup e ripristino database con Dropbox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/dropbox.png</normaloff>:/images/dropbox.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="5" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_up_dw">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Scarica fatture</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="save_fatture">
          <property name="text">
           <string>Scarica fatture</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ripristino DB</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="scarico_file">
          <property name="text">
           <string>Rispristino database</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="connetti">
          <property name="text">
           <string>Connetti</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Backup DB</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="carico_file">
          <property name="text">
           <string>Backup database</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Connetti al client Dropbox</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Carica fatture</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="up_fatture">
          <property name="text">
           <string>Carica fatture</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="esci">
       <property name="text">
        <string>Esci</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTextEdit" name="test_agg">
     <property name="minimumSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>234</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>234</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>connetti</tabstop>
  <tabstop>carico_file</tabstop>
  <tabstop>scarico_file</tabstop>
  <tabstop>esci</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
